src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -41,8 +42,14 @@ public void AddTrace(string trace) =>
 
     public LogLevel LogLevel { get; set; } = LogLevel.Debug;
 
+    internal TaskCompletionSource Completed { get; set; } = new(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
Alternatively, this could be made public and the `Done` func could enforce passing a value. That being said, with the current approach, the call to `.Done` is not even necessary anymore.